app/javascript/functions/ApiUtils.jsx

```diff
@@ -10,5 +8,45 @@ export const signUpUser = user => {
         body: JSON.stringify(user)
     }).then(response => {
         response.json();
+        if (response.status === 204 || response.status === 200){
+            redirect.push({
+                pathname: "/home", 
+                state: {cur_usr_username: user.username}
+            })
+        };
+    });
+}
+
+export const loginUser = (user, redirect) => {
+    fetch('/login_user', {
+        headers: {
+            'Accept': 'application/json',
+            'Content-Type': 'application/json'
+        },
+        method: "POST",
+        body: JSON.stringify(user)
+    }).then(response => {
+        response.json();
+        if(response.status === 204 || response.status === 200){
+            redirect.push({
```

Comment:
Let's not handle the redirects in this file. We can remove it from now and implement a different workflow for redirects in a following PR.